runtime/sunvmi/sunvmi.c

```diff
@@ -90,7 +90,12 @@ latestUserDefinedLoaderIterator(J9VMThread * currentThread, J9StackWalkState * w
 	J9Class * currentClass = J9_CLASS_FROM_CP(walkState->constantPool);
 	J9ClassLoader * classLoader = currentClass->classLoader;
 
-	if (classLoader != vm->systemClassLoader) {
+	/* Ignore jdk/internal/loader/ClassLoaders$PlatformClassLoader (Java 9+).
+	 * In Java 9+, vm->extensionClassLoader is the PlatformClassLoader.
+	 */
+	BOOLEAN isPlatformClassLoader = (J2SE_VERSION(vm) >= J2SE_19) ? (classLoader == vm->extensionClassLoader) : FALSE;
```

Comment:
Have you checked the behaviour on JDK 8?  Should the iterator be skipping the extensionLoader in Java 8?